### PR TITLE
Added some writing guidelines

### PR DESCRIPTION
**Pull request by Uma-95**
@beckermr @kathatherine this is my first PR ever! I would love to have it reviewed.
I added some guidelines on how to contribute to the docs.

### CODE REVIEW

File: src/user/contributing.rst [hunk 1]
```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+   Writing guidelines:
+   1. Identify your audience and Understand their skill level.
```
Comment (ssurbhi560) [inline src/user/contributing.rst:53]: It should be "understand" ( 'u' should be in small case)

File: src/user/contributing.rst [hunk 2]
```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+   2. Match the technical language with to the audience's skill level proficiency.
```
Comment (ssurbhi560) [inline src/user/contributing.rst:54]: "with to the audience", should be "with the audience".

File: src/user/contributing.rst [hunk 3]
```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+   3. Try to Keep it simple.Incase,if you have to use a lot of complicated terms provide a glossary of key terms
```
Comment (ssurbhi560) [inline src/user/contributing.rst:55]: 1. Space after full stop and comma.
2. K in Keep should be in small case.
3. 'In' and 'case' are two words so please consider adding a space between.
4. Also a full stop at the end of this sentence.

File: src/user/contributing.rst [hunk 4]
```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+   4. Address the audience directly as user(s).
+   5. While mentioning about an action, use the "command" form of the verb: "Choose an option from the menu and press Enter."
```
Comment (ssurbhi560) [inline src/user/contributing.rst:57]: It would nice to mention that this is an example.

File: src/user/contributing.rst [hunk 5]
```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
...
+   6. For reference,Provide links to related content.
```
Comment (ssurbhi560) [inline src/user/contributing.rst:58]: P should be small case in Provide, and also space after comma.

File: src/user/contributing.rst (hunk 4 above)
Comment (kathatherine) [inline src/user/contributing.rst:57]: Remove "about" from item 5.

File: src/user/contributing.rst (hunk 5 above)
Comment (kathatherine) [inline src/user/contributing.rst:58]: 1. Make "reference" plural: "references"
2. You need a space after the comma.
3. "provide" should be lowercase.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Congratulations on your first PR! This is a good start, though I agree with Matt that it needs more content.

File: src/user/contributing.rst (hunk 5 above)
Comment (Uma-95) [reply src/user/contributing.rst:58]: Thank you, @ssurbhi560 for those changes. I'll make sure that I will not repeat such silly mistakes hereafter. Also, I'll make the necessary changes and submit the PR again.

Comment (Uma-95) [reply src/user/contributing.rst:58]: Thank you, @kathatherine for those changes. I'll make sure that I will not repeat such silly mistakes hereafter. Also, I'll make the necessary changes and submit the PR again.

**Pull request closed without merging**